plugins/modules/k8s_exec.py

```diff
@@ -144,7 +144,7 @@ def argspec():
 def execute_module(module, k8s_ansible_mixin):
 
     # Load kubernetes.client.Configuration
-    api = core_v1_api.CoreV1Api()
+    api = core_v1_api.CoreV1Api(k8s_ansible_mixin.client.client)
```

Comment:
Could you please add a changelog and test for this change? Thanks,

Reply:
Hi @Akasurde, The change will come when the turbo mode will be merged. Regarding the changelog, this modification is actually a fix of the large pre-turbo mode refactoring, which is still unreleased.